### DISCUSSION TITLE
In the importer, it's not possible to ignore itself with a unique column.

### DISCUSSION THREAD

**Discussion by bzy107** [filamentphp/filament#10756] [Ideas]
I'm having trouble implementing something like `->unique(ignoreRecord: true)` in the importer.
I would appreciate it if you could implement this, or if there's another way to achieve it, please let me know.

**Comment by danharrin** [comment]
`->rules(fn ($record) => [Rule::unique('users')->ignore($record->id)])`

**Reply by bzy107** [reply]
Thank you for your response. 
I also tried this, but it didn't work for me either. I tried it again today, but the result was the same.
Have you tried this?

**Reply by danharrin** [reply]
Please open an issue with a reproduction repo if this doesnt work

**Reply by bzy107** [reply]
It seems it wasn't working well due to another issue. When I tried it again, it worked fine. Thank you
